index.html

```diff
@@ -1391,7 +1391,11 @@ <h2>Core Vocabulary Definitions</h2>
           containing a combination of these terms.</span></p><p class="note">
 		  It is considered to be good practice that each <code>observeproperty</code> has a corresponding
 		  <code>unobserveproperty</code> unless the protocol supports implicit unsubscription mechanisms
-		  (e.g., heartbeat to detect connection loss).</p></section>
+		  (e.g., heartbeat to detect connection loss).</p><p class="note">
+		  The observation mechanism depends on the underlying protocol or sub-protocol.  It is not guaranteed that the latest <a>Property</a> value will be provided when 
+          the subscription is initiated. To observe the latest <a>Property</a> value, it is recommended to first read the <a>Property</a> (<code>readproperty</code>) 
+          and then perform a <a>Property</a> subscription (<code>observeproperty</code>). 
+          </p></section>
```

Comment:
May I propose a simplification that leaves it open when the value is read (before/after)

The observation mechanism depends on the underlying (sub-)protocol. Having said that, it is not guaranteed that the current  <a>Property</a> value will be provided once the subscription is initiated. Hence, it may be necessary to read the current <a>Property</a> value before/after the subscription to get a first value.


@relu91 ?

Reply:
would be also ok for me